sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/Bindings/ServiceBusTriggerAttributeBindingProviderTests.cs

```diff
@@ -60,6 +64,43 @@ public async Task TryCreateAsync_DefaultAccount()
             Assert.NotNull(binding);
         }
 
+        [Test]
+        public async Task GetAutoCompleteMessagesOptionToUse_AutoCompleteDisabledOnTrigger()
```

Comment:
Would be great to get full coverage on all 4 combinations of settings here or in the live tests.